inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1,120 +1,236 @@
+# linelist_raw <- rio::import("linelist.xlsx", which = "Sheet1")
+#
+# Stata data file
+# linelist_raw <- rio::import("linelist.dat")
+#
+# For password protected Excel file 
+# use the excel.link package 
+# library(excel.link)
+# linelist_raw <- excel.link::xl.read.file("linelist.xlsx",
+#                                          xl.sheet = "Sheet1",
+#                                          password = askpass::askpass(prompt = "please enter file
+#                                                                      password"))
+
+# linelist_raw <- outbreaks::fluH7N9_china_2013
+linelist_raw <- gen_data(dictionary = "Mortality", varnames = "column_name",
+                         numcases = 500)
+
+study_data_cleaned <- linelist_raw %>%
   # some cleaning
-  mutate(age_in_years = as.integer(age),
-         age_group = cut(age_in_years, breaks = seq(0, 90, 5), right = FALSE),
-         sex = gender,
-         symp_Tuberculosis = runif(n()) < 0.5,
-         symp_Cancer = runif(n()) < 0.01,
-         symp_Mumps = runif(n()) < 0.1,
-         died = outcome == "Death") %>%
+  mutate(age_in_years = as.integer(q155_q5_age_year),
+         age_group = age_categories(age_in_years, breakers = seq(0, 85, 5)),
+         sex = q4_q6_sex,
+         died = q136_q34_died == "Yes",
+         cause_of_death = factor(q138_q36_died_cause,
+                                 levels = c("Malaria/fever", "Diarrhoea", "Respiratory",
+                                            "Trauma/accident", "Pregnancy-related", 
+                                            "Violence", "Outbreak disease (specify)", 
+                                            "Malnutrition", "Unknown", "Other (specify)"))) %>%
   filter(!is.na(sex))
 ```
 
 
 # Introduction
 
-# Study Design 
-
-# Sample Size and Sampling methods
-
-## Sample Size
-
-```{r}
-size <- sample_size(population_size = 10000,
-                    precision = 0.005, 
-                    design_effect = 1.5, 
-                    expected_prevalence = 0.005)
-size
-```
+# Methods
 
-```{r}
-sample_size_households(size, 
-                       avg_hh = 5, 
-                       prop_under_5 = 0.15, 
-                       frac_6_59 = 0.9, 
-                       non_response_rate = 0.03)
+## Study Design 
 
-```
+## Sample Size and Sampling methods
 
+### Sample Size
 
-## Sampling
+### Sampling
 
-### Random sampling
+#### Random sampling
 
 ```{r}
 # Insert some code to generate a random number sequence
 ```
 
-### Cluster sampling
+#### Cluster sampling
 
 ```{r}
 # Insert some example code to do cluster sampling
 ```
 
-# Study Findings
+## Data collection
+
+## Data analysis
+<!-- TODO: add paragraph about R/sitrep -->
+
+# Results
 
 
 ```{r}
 # we first need to create a survery design that reflects the study
 suvery_design <- svydesign(
   ids = ~1, # no cluster ids
-  weights = NULL, #no sampling weights in this example
+  weights = NULL, # no sampling weights in this example
```

Comment:
Think we should have two examples here for survey_design. Once for simple random and once for cluster. 
To do this you need to gen population by age and sex (see cholera template for making fake population). 
Have forwarded you an email with explanation and example population dataset. (basically you merge and multiply..) 

In the second example for cluster sampling - household ID variable is put in the ids option above. (Do this in a second step commented out - just so people see how it would be done). You may need to merge the two different sheets from the survey excel (one is household one is individual) - see @pbkeating  pull request from vaccination survey on an explanation and code for that.

Reply:
minor note: `suvery_design` should be `survey_design`